### PR TITLE
[WIP] implement rawOptions

### PR DESCRIPTION
**Pull request by eyalpost**
Parse all options into a rawOptions array. Repeated options will appear multiple times in that array

### CODE REVIEW

**Comment by eyalpost** [comment]
Solves #1142

File: tests/comp_options-textformat.js [hunk 1]
```diff
@@ -15,13 +15,41 @@ message Test {\
...
+}\
+message TestRepeated {\
+  string value = 1 [(my_options) = { a: \"foo\" a:\"foo2\" b: \"bar\" } , (my_options) = { a: \"foo3\" a:\"foo4\" b: \"bar2\" }];\
+  string value2 = 2 [(my_options) = { a: \"foo\" b { c: \"bar\" }, b { c: \"bar2\"} }];\
+  string value3 = 3 [(my_options) = { a: \"foo\", b: \"bar\", b: \"bar2\" }];\
+}\
+";
 
-tape.test("options in textformat", function(test) {
+tape.only("options in textformat", function(test) {
```
Comment (hugebdu) [inline tests/comp_options-textformat.js]: `only` you

Comment (eyalpost) [reply tests/comp_options-textformat.js]: I was surprised all tests passed ;-)

**Comment by eyalpost** [comment]
Don't merge - I'm adding another scenario

**Comment by eyalpost** [comment]
Different approach https://github.com/protobufjs/protobuf.js/pull/1256

**Pull request closed without merging**